privilege-management.md

```diff
@@ -192,9 +192,17 @@ Dynamic privileges include:
 
 * `BACKUP_ADMIN`
 * `RESTORE_ADMIN`
+* `SYSTEM_USER`
+* `SYSTEM_VARIABLES_ADMIN`
 * `ROLE_ADMIN`
 * `CONNECTION_ADMIN`
-* `SYSTEM_VARIABLES_ADMIN`
+* `PLACEMENT_ADMIN` allows privilege owners to create, modify, and remove placement policies.
+* `DASHBOARD_CLIENT` allows privilege owners to log in to the TiDB-Dashboard.
+* `RESTRICTED_TABLES_ADMIN` allows privilege owners to see system tables when SEM is enabled.
+* `RESTRICTED_STATUS_ADMIN` allows privilege owners to see all status vars when SEM is enabled.
+* `RESTRICTED_VARIABLES_ADMIN` allows privilege owners to see all variables when SEM is enabled.
+* `RESTRICTED_USER_ADMIN` prohibits privilege owners to have their access revoked by SUPER users.
+* `RESTRICTED_CONNECTION_ADMIN` prohibits privilege owners to have connections killed by the `PROCESS` or `CONNECTION_ADMIN` privilege.
```

Comment:
More details in the RFC: 
Killing connections that belong to your own user is always permitted. Killing connections that belong to other users requires `CONNECTION_ADMIN` or `SUPER`; but there is an exception for `RESTRICTED_USER_ADMIN` users. **To kill these connections also requires `RESTRICTED_CONNECTION_ADMIN`**. This affects `KILL`, `KILL TIDB` commands.